docs/farming-&-staking/farming/intro.mdx

```diff
@@ -121,17 +122,17 @@ Only necessary if your system has limited RAM.
 
 :::info Safety and Security Guide
 
-For detailed information, visit our [Safety and Security](../../learn/security) guide.
+For detailed information, visit our [Safety and Security](/learn/security) guide.
 
 :::
 
-Ensure your system is regularly updated, configure [port forwarding](additional-guides/port-config.mdx#ports-to-forward--firewall-access), set up your [firewall](additional-guides/port-config.mdx#firewall-configuration), and follow best practices for network safety. For comprehensive security guidelines, please refer to our [Safety and Security](../../learn/security) guide.
+Ensure your system is regularly updated, configure [port forwarding](additional-guides/port-config.mdx#ports-to-forward--firewall-access), set up your [firewall](additional-guides/port-config.mdx#firewall-configuration), and follow best practices for network safety. For comprehensive security guidelines, please refer to our [Safety and Security](/learn/security) guide.
```

Comment:
Pretty sure it's required
```suggestion
Ensure your system is regularly updated, configure [port forwarding](/additional-guides/port-config.mdx#ports-to-forward--firewall-access), set up your [firewall](/additional-guides/port-config.mdx#firewall-configuration), and follow best practices for network safety. For comprehensive security guidelines, please refer to our [Safety and Security](/learn/security) guide.
```

Reply:
We can abstract away the file name in the URL by using slugs, making it look like a path rather than a file. Anchors work with slugs without requiring the filename and extension. This approach will also reduce the need to add redirects in the future if the filename is changed or moved.